src/main/java/frc/robot/subsystems/Arm.java

```diff
@@ -0,0 +1,40 @@
+package frc.robot.subsystems;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.TalonFX;
+
+public class Arm {
+    public final class Constants{
```

Comment:
You have the right idea here. The way it is right now, it would work, however we have a file called Constants.java where we want to put the constants for every subsystem. You can rename Constants to ArmConstants, and copy everything here into Constants.java